docs/data/date-pickers/overview/CommonlyUsedComponents.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import Tooltip from '@mui/material/Tooltip';
+import Stack from '@mui/material/Stack';
+import { DemoContainer, DemoItem } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { DateRangePicker } from '@mui/x-date-pickers-pro/DateRangePicker';
+
+function Label({
+  componentName,
+  valueType,
+  isProOnly,
+}: {
+  componentName: string;
+  valueType: string;
+  isProOnly?: boolean;
+}) {
+  const content = (
+    <span>
+      <strong>{componentName}</strong> for {valueType} editing
+    </span>
+  );
+
+  if (isProOnly) {
+    return (
+      <Stack direction="row" spacing={0.5} component="span">
+        <Tooltip title="Included on Pro package">
+          <a href="/x/introduction/licensing/#pro-plan">
+            <span className="plan-pro" />
+          </a>
+        </Tooltip>
+        {content}
```

Comment:
I'd suggest inverting the order of text and icon to align with other usages.
```suggestion
        {content}
        <Tooltip title="Included in Pro package">
          <a href="/x/introduction/licensing/#pro-plan">
            <span className="plan-pro" />
          </a>
        </Tooltip>
```

Reply:
@joserodolfofreitas made me reverse in this way :laughing:

Reply:
🙈